charts/gardener-extension-validator-vsphere/charts/runtime/templates/deployment.yaml

```diff
@@ -46,6 +46,19 @@ spec:
         {{- if .Values.global.kubeconfig }}
         - --kubeconfig=/etc/gardener-extension-validator-vsphere/kubeconfig/kubeconfig
         {{- end }}
+        - --health-bind-address=:{{ .Values.global.healthPort }}
```

Comment:
It makes sense to wrap this in case the values are not provided, see https://github.com/gardener/gardener/blob/e8e73842ad0acfe30c02eb7d17c8c7b2fd42a4e5/charts/gardener/gardener-resource-manager/templates/deployment.yaml#L62-L64

Reply:
It was suggested here not to add if https://github.com/gardener/gardener-extension-provider-openstack/pull/421#discussion_r835330230